rtptransceiver.go

```diff
@@ -92,6 +101,10 @@ func (t *RTPTransceiver) setReceiver(r *RTPReceiver) {
 }
 
 func (t *RTPTransceiver) setDirection(d RTPTransceiverDirection) {
+	// https://www.w3.org/TR/webrtc/#ref-for-dfn-stopping-4
+	// if stopping {
```

Comment:
should return an error here according to spec but requires quite a lot of changes to the code base. not sure if we should update?